freqtrade/exchange/__init__.py

```diff
@@ -70,6 +70,9 @@ def __init__(self, config: dict) -> None:
         # Check if all pairs are available
         self.validate_pairs(config['exchange']['pair_whitelist'])
 
+        # Check if timeframe is available
+        self.validate_timeframes(config['ticker_interval'])
```

Comment:
This line fails if a config without ticker_interval is used (because it's configured in the strategy ...)